src/main/java/com/arpnetworking/metrics/mad/Aggregator.java

```diff
@@ -342,6 +461,17 @@ public Builder setPeriods(final Set<Duration> value) {
             return this;
         }
 
+        /**
+         * Set the idle timeout for workers. Cannot be null.
```

Comment:
Maybe add a comment here about the actor reaping occurring when the timeout is reached.

Reply:
Done!